dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -17,12 +17,11 @@ IBM-Process-Types: client, \
  server
 Subsystem-Name: Java XML Bindings 2.3
 -features=\
-  com.ibm.websphere.appserver.classloading-1.0
+  com.ibm.websphere.appserver.classloading-1.0, \
+  com.ibm.websphere.appserver.internal.optional.jaxb-2.3
 -bundles=\
-  com.ibm.websphere.javaee.activation.1.1; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.activation:activation:1.1", \
-  com.ibm.websphere.javaee.jaxb.2.3; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.xml.bind:jaxb-api:2.3.0", \
-  com.ibm.ws.org.apache.geronimo.osgi.registry.1.1, \
-  com.ibm.ws.jaxb.tools.2.3
+  com.ibm.websphere.javaee.activation.1.1; location:="dev/api/spec/,lib/", \
```

Comment:
why were the `mavenCoordinates` dropped on this line and the next line? They should be added back

Reply:
mavenCoordinates are added back